src/actions/actionCreators.js

```diff
@@ -0,0 +1,13 @@
+import {
+  SOCIAL_LOGIN_INITIATED,
```

Comment:
@kakaemma I like the one import per line convention you are using here! And making use of an `actionCreators` file